api/test/test_helper/fake_android_plugin.gd

```diff
@@ -65,3 +75,37 @@ func getPackLocation(pack_name : String):
 	if pack_name in _asset_pack_location_store:
 		return _asset_pack_location_store[pack_name]
 	return null
+
+func getPackLocations():
+	return _asset_pack_location_store
+
+# -----------------------------------------------------------------------------
+# If on_cancel_return_not_found is false, returns an AssetPackStates object 
+# where all updated AssetPackState will be instantiated by duplicating
+# _asset_pack_state_on_cancel. Else returns an Dictionary that simulates 
+# the situation where user cancels a nonexisting asset pack.
+# -----------------------------------------------------------------------------
+func cancel(pack_names : Array):
```

Comment:
Rather than having an extra bool to return not found, could you instead maintain a PlayAssetPackStates `var` for "active downloads" and have this method check if the specified pack_names is present in that PlayAssetPackStates? That way you get a little closer to simulating the plugin and you can leverage that PlayAssetPackStates later on when you add `fetch` (i.e. calling `fetch` adds a PlayAssetPackState).